frontend/talentsearch/src/js/components/Browse/BrowsePoolsPage.tsx

```diff
@@ -278,10 +278,10 @@ export const BrowsePools: React.FC<BrowsePoolsProps> = ({
                 external: true,
                 label: intl.formatMessage({
                   defaultMessage:
-                    "<hidden>Learn more about </hidden>Indigenous IT apprenticeship<hidden> opportunities</hidden>",
-                  id: "xgocDD",
+                    "Apply<hidden> to the Indigenous Apprenticeship Program</hidden> now",
```

Comment:
The leading space inside the hidden tags seems to get chomped in Firefox.
![image](https://user-images.githubusercontent.com/8978655/200331267-aeff10f7-c3e9-4630-b88c-91c96ec6224f.png)
When I changed it to `"Apply <hidden>to the Indigenous Apprenticeship Program </hidden>now"`
![image](https://user-images.githubusercontent.com/8978655/200330600-47313a17-ea40-4e99-b89b-058571afcbf7.png)
I'm not sure if this is the best solution or if it works the same way in different browsers.

It looks like we may have this issue in a number of places.
![image](https://user-images.githubusercontent.com/8978655/200331709-bcdd1bcc-b189-42c6-b07f-8eadba59799e.png)

Reply:
Good catch @petertgiles. This indeed is an issue in other places as well in Firefox (but not Chrome).

### Screenshots

![Screen Shot 2022-11-07 at 09 58 42](https://user-images.githubusercontent.com/3046459/200343612-adfcff01-3e48-4eb0-ae8c-63860e135b48.png)
![Screen Shot 2022-11-07 at 09 56 12](https://user-images.githubusercontent.com/3046459/200343615-eff970ac-7856-42b7-9c5a-b557eb00ed8a.png)

Strangely, this similar string does not have these same problems in Chrome or Firefox:

https://github.com/GCTC-NTGC/gc-digital-talent/blob/8d57e6e41f7a3afc98c40703000f30878642a93c/frontend/talentsearch/src/js/components/employmentEquityForm/EquityOption.tsx#L34

@esizer is it because the text within the hidden tag cannot begin with a space? Is this a known Firefox bug?

Reply:
I've created a new issue for the other strings since those are out of scope: https://github.com/GCTC-NTGC/gc-digital-talent/issues/4619

Reply:
My best guess is that since it is a `span` and by default has `display: inline;` and Firefox will collapse whitespace inside an inline element.

> And finally, sequences of spaces at the beginning and end of an element are removed...

REF: https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Whitespace#:~:text=And%20finally%2C%20sequences%20of%20spaces%20at%20the%20beginning%20and%20end%20of%20an%20element%20are%20removed%2C%20so%20we%20finally%20get%20this%3A

Reply:
I tried this `Apply <hidden>to the Indigenous Apprenticeship Program </hidden>now` and it is formatted correctly in the a11y inspector but for some reason, VoiceOver reads it in the wrong order as: **to the indigenous Apprenticeship Program Apply now** 😕. Any ideas?

![Screen Shot 2022-11-07 at 10 32 52](https://user-images.githubusercontent.com/3046459/200351449-b9ab6b7a-c371-4b7d-b1cc-ff7aa19943b8.png)

Reply:
Oof, I'm guessing this is also part of a much larger issue (and scope).  I'd love to get this PR merged it to green up our CI.  Thoughts on spinning off this issue and merging the PR as is?

Reply:
Yes, it is definitely part of a larger issue. Let's go green and worry about it as part of: 

- https://github.com/GCTC-NTGC/gc-digital-talent/issues/4619

Reply:
Looks like this is a [known issue](https://github.com/h5bp/main.css/issues/12) with VoiceOver when `margin: -1;`....

Seems like [it is required](https://github.com/twbs/bootstrap/issues/25686) though, so it may be out of our control until VoiceOver has a fix for this.